### ISSUE TITLE
Errors on ios

### ISSUE THREAD

**Issue by dvcrn** [OssamaZ/react-native-flux-boilerplate#3]
Hey, cool boilerplate! It saved me a good amount of work :) 

I noticed that a few things are not working on iOS. Maybe you can give some details in how it is supposed to work? 

The webpack bundler runs on 8082 and the server on 8080. When trying to pull `index.ios.bundle` from `8080`, I'm getting 

```
<html> 
<head><title>404 Not Found</title></head> 
<body bgcolor="white"> 
<center><h1>404 Not Found</h1></center> 
<hr><center>nginx/1.8.0</center> 
</body> 
</html> 
```

When I change the URL to `index.ios.js` on 8082, I'm getting `Can't find variable navigator`. Did I break something?

**Comment by dvcrn** [comment]
Oh whoops, looks like I didn't read correctly. It is supposed to start inside `App/`. 
In that case codeLocation still should be something like:
```
  jsCodeLocation = [NSURL URLWithString:@"http://localhost:8081/App/index.bundle"];
```

**Comment by dvcrn** [comment]
Should have read the error message better, it clearly states nginx. 
Looks like I had nginx installed which I completely forgot about and ran on 8080. Since I didn't get any errors on the command line, I expected everything to be fine. 

tl;dr all this debugging to realise nginx was running on the same port. 

Great boilerplate!
